tools/scripts/update-docker-tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -ex
+# Retags all docker images from latest Environments build
+# tools/scripts/update-docker-tags.sh NEW_VERSION [--release]
+
+if [ "$#" -lt 1 ] || [ "$#" -gt 1 ] || [ "$#" -eq 2 ] && [ "$2" != "--release" ]; then
```

Comment:
Should those last two conditions be combined with a parenthetical?  The precedence may work here, but I'm not super confident and it's a little tough to know the intent for sure.

Reply:
changed this conditional completely